### DISCUSSION TITLE
Practica 1 - Toma de Requerimientos

### DISCUSSION THREAD

**Discussion by cartovar** [Launch-X-Latam/MisionFrontEnd#143] [Q&A]
Un saludo Explorer, en esta oportunidad me acerco con el fin de poder aclarar las siguientes dudas en relación a la toma de requerimientos:

Este documento se encuentra dividido en 5 partes:

1- Descripción General de Requerimiento (COMPLETADO)
2- Fase de Formalización (COMPLETADO)
3- Análisis de Requisitos y Requerimientos (COMPLETADO)
    NOTA:
    3.1 el Diagrama BPMN lo hice en https://demo.bpmn.io/new
    3.2 Plan de trabajo en conjunto con las estimaciones lo pude realizar en una plantilla predeterminada de Excel.

Mis dudas van en referencia a los puntos:

4- Levantamiento de Requerimiento Detallado
5- Diseño de la Arquitectura de la Solución

En estos dos últimos puntos no comprendo exactamente que hay que detallar o si esta parte del documento se va completando a medida que vayamos avanzando en el BootCamp. Solicito de su ayuda para poder aclarar estas inquietudes.

Para el resto de las actividades relacionadas,

Buyer Persona (COMPLETADO)
Publico Objetivo (COMPLETADO)
Crea tu primer Wireframe UX (EN PROCESO)
UI (PENDIENTE)

Muchas gracias de antemano por cualquier ayuda que me puedan brindar.

**Comment by doguedogue** [comment]
Este punto: 4- Levantamiento de Requerimiento Detallado, está relacionado al punto 1- Descripción General de Requerimiento, pero con mas detalle, explicando cada una de las cosas, para que a la hora de diseñarlo sea más fácil.

Este punto depende obviamente del punto 4, 5- Diseño de la Arquitectura de la Solución, donde defines como va a ser la solución del requerimiento. Existen muchas arquitecturas, define una que gustes, ejemplos: Web, con alta disponibilidad, con un servidor, MVC, etc, date gusto,  ponte el sombrero de que eres un freelancer vendiendo una página web ó app ó lo que pienses hacer.

**Reply by cartovar** [reply]
Muchas gracias por aclarar las dudas

**Reply by DaviiidSG** [reply]
Hola, podrían echarme una mano con la parte de toma de requerimientos? La verdad no entiendo del todo como llenar ese formulario y me vendría excelente si me ayudaran o me aconsejaran ya que es lo único que me falta. Gracias de antemano
